src/AppBundle/Entity/Ndr/BankAccount.php

```diff
@@ -24,12 +26,18 @@ class BankAccount implements BankAccountInterface
         'other_no_sortcode',
     ];
 
+    /**
+     * @JMS\Exclude
+     */
     private static $typesNotRequiringSortCode = [
         'postoffice',
         'cfo',
         'other_no_sortcode'
     ];
 
+    /**
+     * @JMS\Exclude
+     */
```

Comment:
Are these exclusions not used?

Reply:
They're needed, otherwise JMS Serializer throws an exception when it tries to access them (which was breaking the page render). This has [already been done on the API](https://github.com/ministryofjustice/opg-digi-deps-api/blob/master/src/AppBundle/Entity/Ndr/BankAccount.php#L20) but I guess has gone unnoticed on client because my code deserialises differently to existing functionality (e.g. I retrieve a bank account directly, not as part of a report).